Assets/Vimeo/Scripts/Recorder/RecorderController.cs

```diff
@@ -128,13 +128,13 @@ public void EndRecording()
 
             if (videoInput != null) {
                 videoInput.EndRecording();
-                 Destroy(videoInput);
+                 DestroyImmediate(videoInput);
```

Comment:
Worth reading this https://answers.unity.com/questions/18651/difference-between-destroy-and-destroyimmediate-.html ... Seems like `Destroy` is preferred, but `DestroyImmediate` is required for Editor only tests. Perhaps the solution is to create a new destroy method in this class that detects if in Unity editor mode vs not?